lib/sles4sap.pm

```diff
@@ -347,26 +342,10 @@ sub copy_media {
     assert_script_run "mkdir $target";
     assert_script_run "mount -t $proto -o ro $path $mnt_path";
     $media_path = $mnt_path if script_run "[[ -d $media_path ]]";    # Check if specific ARCH subdir exists
-    assert_script_run "cp -ax $media_path/. $target/", $nettout;
+    assert_script_run "rsync --archive --checksum $media_path/. $target/", $nettout;
```

Comment:
Yes setting a rsync server on the share server (don't remember his name) and remove NFS in that test could be helpful if you have NFS timeout issues. There wasn't timeout issue when I created the server some years ago, but AFAIR the VM was migrated to another hypervisor and this one look either less powerful or is too busy. So clearly native rsync with retry could help.

Just keep in mind to keep NFS share and use NFS protocol in the YaST SAP Wizard test, as it could be use by customer (and also other protocols, like SMB, but I didn't had time in the past to add them, was not high prio at this time).

Reply:
I've set up an rsync daemon on the same host. So the following works now: `rsync -av rsync://1c119.qa.suse.de/sap/HANA2/SPS04rev46/x86_64/ .`

Reply:
Before fully getting rid of NFS in the module, verify that an rsync server can be configured in the HANA pre-validation openQA instance. Network ACLs there could be different to what we have in osd & R&D network.